content/ui-patterns/messaging.mdx

```diff
@@ -0,0 +1,71 @@
+- [Flash Alerts](https://primer.style/css/components/alerts)
+- [Popovers](https://primer.style/css/components/popover)
+
+## Messaging Types
+It's important to consider the context when selecting a component to convey your message. Does the message need to interrupt the user flow? Is this a global message that needs to be shown on every page? Does it require the user to take an action, or is it more of a notice? Most messaging on GitHub falls into one of the following categories:
+
+**Feedback**: communicating something to the user based on an action they've taken or a change in context
+- _Toasts or Flash Alerts are best suited for this type of messaging._
+
+**Highlights**: bringing attention to how and why to engage with a feature
+- _Popovers are best suited for this type of messaging._
+
+**Updates**: communicating necessary updates or changes to the product as it relates to the user
+- _Flash Alerts are best suited for this type of messaging._
+
+## Toasts
+Toasts are used to provide relevant feedback to the user, typically after they've taken an action. Primer includes styles for success, warning, error, loading, and default Toasts. Some examples of usage include:
+
+- confirmation that an action was successfully taken
+- communicating that an action is pending
+- notifying the user if an action failed to complete.
+- providing general information.
+
+Toasts are best used in context of the page they're referring to (rather than a global notification that can appear on any page) and should require minimal user interaction. For global notices and messaging, see the Flash component.
+
+Toasts should be brief and not bog down the experience with superfluous copy. If multiple toasts appear on the page, they will stack naturally.
+
+### Accessibility
+Toasts are non-modal components and should contain `role=log` , which implies the element has `aria-live="polite"`. This notifies the user of the Toast via Assistance Technologies without having to change focus to the Toast. You can read more about `role=log` [here](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA23).
+
+Toasts are generally informative and should not receive focus when they appear. For that reason, we suggest you **avoid using interactive elements** in the component (aside from a dismiss action). Keep in mind that, even without an explicit dismiss action, the user can always hit `esc` to dismiss the Toast. For more information on how interactive children affect web accessibility, [check out this issue](https://github.com/jackbsteinberg/std-toast/issues/29).
+
+
+## Flash Alerts
+Flash alerts are used to display updates or alerts pertaining to any part of the system; this includes information regarding the user's account, the organization, the repo, and more. These updates are typically not user-initiated but rather alerts that require the user's attention.
+
+Similarly to Toasts, Flash Alerts can convey a warning, error, success, or a neutral message.
```

Comment:
Should we capitalize here? Also I believe we only use sentence case for UI. [Brand guidelines](https://brand.github.com/content/grammar-usage/#headers-and-ui-elements) for reference.